week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,13 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+var students = 104;
+var mentors = 56;
+
+var total = students + mentors;
+
+  var percentMentors = Math.round((mentors *100)/total);
+
+  var percentstudents = Math.round((students *100)/total);
+function message (){
+    var gM = "percentage of mentors is " + percentMentors+ "%" + " and percentage of students is " + percentstudents + "%";
+    return gM;
+}
```

Comment:
Good program! 🎉 
A couple of things that can be improved:
- how about renaming some of those variables? :D
- generally, variable names are nouns, and function names include a verb. How can you rename the `message()` function then?
- be careful with formatting - the `()` shouldn't have any spaces in front of them

Reply:
@martynamaron, I have changed the variable names  and I have renamed the message as classMentorGreeting